app/src/main/java/com/gh4a/activities/PullRequestActivity.java

```diff
@@ -167,6 +173,22 @@ protected void onResultReady(Boolean result) {
         }
     };
 
+    private final LoaderCallbacks<Reference> mHeadReferenceCallback = new LoaderCallbacks<Reference>(this) {
```

Comment:
I wonder whether we could/should do this after loading the PR so we can re-use the loaded PR data and avoid calling getPullRequest() twice. We could still show the options menu and just hide the item until the ref is loaded.

Reply:
Updated.